src/Message.php

```diff
@@ -152,7 +152,7 @@ public function getHeaders(): Message\Headers
             // imap_headerinfo is much faster than imap_fetchheader
             // imap_headerinfo returns only a subset of all mail headers,
             // but it does include the message flags.
-            $headers = \imap_headerinfo($this->resource->getStream(), \imap_msgno($this->resource->getStream(), $this->getNumber()));
+            $headers = imap_rfc822_parse_headers(\imap_fetchheader($this->resource->getStream(), \imap_msgno($this->resource->getStream(), $this->getNumber())));
```

Comment:
In my tests I see no error:
```
Ddeboer\Imap\Message\Headers Object
(
    [storage:ArrayIterator:private] => Array
        (
            [date] => Wed, 13 Sep 2017 13:05:45 +0200
            [subject] => Nuu
            [toaddress] => to1 <to1@here.com>, to2 <to2@here.com>
            [to] => Array
                (
                    [0] => stdClass Object
                        (
                            [personal] => to1
                            [mailbox] => to1
                            [host] => here.com
                        )

                    [1] => stdClass Object
                        (
                            [personal] => to2
                            [mailbox] => to2
                            [host] => here.com
                        )

                )
    [...]
```
Could you please post here the raw message that originated for you the issue?